enterprise/models.py

```diff
@@ -347,7 +347,7 @@ def identity_providers(self):
     @property
     def has_identity_providers(self):
         """
-        Return the unique slug for the identity provider associated with this enterprise customer.
+        Return the identity providers associated with this enterprise customer.
```

Comment:
good catch, forgot to update this.
I think is should be something like.
"""
Return True if there are any identity providers associated with this enterprise customer.
"""